news/2018-11-21-new-featured-artist-culprate.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New Featured Artist: Culprate"
+date: 2018-11-21 10:00:00 +0000
+---
+
+We proudly welcome an extremely talented producer hailing from the United Kingdom to the osu! Featured Artist lineup — Culprate!
+
+[![](https://assets.ppy.sh/artists/29/header.jpg)](https://osu.ppy.sh/beatmaps/artists/29)
+
+Something of a standout among the UK's underground dubstep scene, **Culprate** is a tremendously capable music producer with some incredible musical clout. Renowned for his incredible production quality and captivatingly complex tracks, we pretty much fell in love in his sound when we first heard it.
```

Comment:
fell in love with

Reply:
it's right, Eph and Culprate fell in love while listening to his music